### DISCUSSION TITLE
vscode debug output

### DISCUSSION THREAD

**Discussion by TheColorRed** [slint-ui/slint#6798] [Q&A] [answered]
When designing the layout, is there a way to view the `debug("xxx")` data? I tried looking a the developer tools in vscode, but nothing was there, so I am not sure if I am able to see debug info or not. 

If this isn't implemented, it would be helpful to have a built-in debug console, similar to jsfiddle. [Example](https://jsfiddle.net/9dch7qub/)

![image](https://github.com/user-attachments/assets/a744d47c-4e48-4b59-957f-024e29a1617a)

**Comment by ogoffart** [comment, marked as answer]
there is a "Slint" output in the Output tab of vscode where to debug shows.
![image](https://github.com/user-attachments/assets/51b2a023-d1b0-40c4-ab6c-0e2d0829daea)


(But it is true it would be nice if it was shown in the preview window itself)
